qpc_tools/messages.py

```diff
@@ -26,6 +26,10 @@
 SERVER_INSTALL_OFFLINE_FILES_HELP = 'Specify the path to the server offline files'
 SERVER_INSTALL_VERSION_HELP = 'Specify the server version to install '\
     '(defaults to latest release)'
+SERVER_INSTALL_SERVER_IMAGE_HELP = 'Sets the name of the server image. Defaults to ' \
```

Comment:
change `Sets` -> `Set`. Also, i don't think it defaults to the `the latest discover-server on redhat.registry.io` for upstream, - I think it defaults to quipucords for upstream?

Reply:
update -> you can leave this in the downstream perspective since that's the only time it will be seen. just fix the typo in `discover-server`